src/components/inscripcionActividad/FinalDDJJ.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Grid from '@material-ui/core/Box';
+import { Typography, Button, useMediaQuery } from '@material-ui/core';
+import WarningIcon from '@material-ui/icons/Warning';
+import PeopleIcon from '@material-ui/icons/People';
+import { makeStyles } from '@material-ui/core/styles';
+import EventAvailableIcon from '@material-ui/icons/EventAvailable';
+import { Link } from 'react-router-dom';
+
+const defaultProps = {
+  //borderColor:"#009688",
+  m: 2,
+  border: 1,
+  style: { width: '40rem', height: '9rem' },
+};
+
+const useStyles = makeStyles({
+  colorTexto: {
+    color: 'white',
+  },
+});
+
+export default function FinalDDJJ() {
+  const classes = useStyles();
+  const matches = useMediaQuery('(min-width:600px)');
+
+  return (
+    <>
+      <Grid direction="column" justifyContent="center" alignItems="center">
+        <Grid mt={matches ? 10 : 4} display="flex" justifyContent="center">
```

Comment:
Creo que la forma de cambiarle el tamaño sin el hook es esta:

```suggestion
        <Grid sm={10} xs={4} display="flex" justifyContent="center">
```

Los breakpoints los lista acá: https://material-ui.com/system/basics/#responsive